packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -160,6 +162,24 @@ export class DocumentDeltaConnection
 					this.emit(event, ...args);
 				});
 			}
+
+			// log latency every minute
+			this.trackLatencyTimer = setInterval(() => {
+				const start = Date.now();
+
+				// emit pong event every minute. If latency is longer than 1 min, log separately
+				this.socket.emit("pong", () => {
+					const latency = Date.now() - start;
+					if (latency > 1000 * 60) {
+						this.mc.logger.sendPerformanceEvent({
+							eventName: "LatencyTooLong",
+							driverVersion,
+							latency,
+						});
+					}
+					return latency;
```

Comment:
I believe this line does not do anything. You need to raise an event instead:
```ts
this.emit("pong", latency);
```

Reply:
Can we add some test coverage here somehow? It was broken before, and it does not work in current stage, so we need to have some test coverage that proves it works / does not work.
I wonder if we can leverage end-to-end tests? I think these changes apply to all but memorylicious driver / service, so maybe we can add end-to-end test that validates we get a pong shortly after connection? And scope this test to exclude  memorylicious  case?
Can you please take a look into it / ask around how to make it happen? I think Andrei did something like that in the past.